src/components/ui/AppCardDrag.vue

```diff
@@ -3,7 +3,7 @@
     <v-card-title
       class="card-title card-heading py-1 draggable"
     >
-      Drag Me
+      {{ $t('app.general.label.drag_me') }}
```

Comment:
I don't see this file being imported anywhere - maybe we should just remove it?

Reply:
Very good point there, I was actually surprised as I didn't remember seeing "drag me" string before... I'll remove the file and translation!

Reply:
Done!